roxie/ccd/ccdmain.cpp

```diff
@@ -1120,7 +1117,7 @@ int STARTQUERY_API start_query(int argc, const char *argv[])
                     unsigned listenQueue = roxieFarm.getPropInt("@listenQueue", DEFAULT_LISTEN_QUEUE_SIZE);
                     unsigned numThreads = roxieFarm.getPropInt("@numThreads", numServerThreads);
                     unsigned port = roxieFarm.getPropInt("@port", ROXIE_SERVER_PORT);
-                    unsigned requestArrayThreads = roxieFarm.getPropInt("@requestArrayThreads", 5);
+                    //unsigned requestArrayThreads = roxieFarm.getPropInt("@requestArrayThreads", 5);
```

Comment:
Should be removed.

Reply:
Should probably actually raise a JIRA to see why it is not being used.